patches/lighthouse+10.4.0.patch

```diff
@@ -0,0 +1,10 @@
+diff --git a/node_modules/lighthouse/core/index.d.cts b/node_modules/lighthouse/core/index.d.cts
+index 1c399e1..ec55bf3 100644
+--- a/node_modules/lighthouse/core/index.d.cts
++++ b/node_modules/lighthouse/core/index.d.cts
+@@ -1,4 +1,5 @@
+ export = lighthouse;
+ /** @type {import('./index.js')['default']} */
++// @ts-ignore
```

Comment:
Perhaps we can add a comment above explaining why we want to ignore it, so that we'll know when we can remove this local patch.